### PR TITLE
fix: styles and toc

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] |
| :--------------------: |

## 🧰 Changes

Fixes rendering the ToC to the right, as well as loads in the main apps bundles.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

**Review by trishaprile — APPROVED** [review]
🎉
image

File: example/index.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <html>
   <head>
     <title>Markdown Demo</title>
-    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/css/bundle-hub2.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/public/hub/web/main.3f4ed9ef7e8b221de43c.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.readme.io/public/hub/web/3435.9200159a8fec79d971c4.css" />
```
Comment (kellyjosephprice) [inline example/index.html:6]: @rafegoldberg is there a better way to pull in general app styles?

Comment (rafegoldberg) [reply example/index.html:6]: Technically we shouldn't need the `bundle-hub2.css` _or_ the React app styles here, but I don't know a better way of including ‘em in the demo app unfortunately, no…

Comment (rafegoldberg) [reply example/index.html:6]: I guess you could move these in to the `demo.scss` as `@import` statements. Still manual, but might _feel_ a little cleaner?

**Review by rafegoldberg — APPROVED** [review]
Fine by me.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.64.0</b>](/readmeio/markdown/releases/tag/v6.64.0)
<br>

**Pull request merged**